packages/sui-hoc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/hoc",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
did you already release it or why is this + changelog here?

Reply:
I'm not sure about this. The only explanation I cant think of regarding this is that i took these changes when merging from master to get the latest changes on this PR